overlord/devicestate/devicemgr.go

```diff
@@ -1027,6 +1066,7 @@ func (m *DeviceManager) RequestSystemAction(systemLabel string, action SystemAct
 		}
 	}
 	if sysAction == nil {
+		// XXX: provide more context here like what mode was requested?
```

Comment:
no strong opinion on this

Reply:
OTOH the caller knows which mode it requested, getting ErrUnsupportedAction means that that combination of system & mode cannot be applied.